modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -0,0 +1,389 @@
+        override: true
+    service:
+      pipelines:
+        traces:
+          processors: [resourcedetection]
+        metrics:
+          processors: [resourcedetection]
+----
+
+=== Exporters
+
+==== OTLP exporter
+
+The OTLP gRPC exporter exports data using the OpenTelemetry protocol (OTLP).
+
+* Support level: link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]
+* Supported signals: traces, metrics
+
+.OpenTelemetry Collector custom resource with an enabled OTLP exporter
+[source,yaml]
+----
+  config: |
+    exporters:
+      otlp:
+        endpoint: tempo-ingester:4317 <1>
+        tls: <2>
+          ca_file: ca.pem
+          cert_file: cert.pem
+          key_file: key.pem
+        headers: <3>
+          X-Scope-OrgID: "dev"
+    service:
+      pipelines:
+        traces:
+          exporters: [otlp]
+        metrics:
+          exporters: [otlp]
+----
+<1> The OTLP gRPC endpoint. If the `+https://+` scheme is used, then client transport security is enabled and overrides the `insecure` setting in the `tls`.
+<2> The client side TLS configuration. Defines paths to TLS certificates.
+<3> Headers are sent for every RPC performed during an established connection.
+
+==== OTLP HTTP exporter
+
+The OTLP HTTP exporter exports data using the OpenTelemetry protocol (OTLP).
+
+* Support level: link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]
+* Supported signals: traces, metrics
+
+.OpenTelemetry Collector custom resource with an enabled OTLP exporter
+[source,yaml]
+----
+  config: |
+    exporters:
+      otlphttp:
+        endpoint: http://tempo-ingester:4318 <1>
+        tls: <2>
+          ca_file: ca.pem
+          cert_file: cert.pem
+          key_file: key.pem
+        headers: <3>
+          X-Scope-OrgID: "dev"
+
+    service:
+      pipelines:
+        traces:
+          exporters: [otlphttp]
+        metrics:
+          expoters: [otlphttp]
+----
+<1> The OTLP HTTP endpoint. If the `+https://+` scheme is used, then client transport security is enabled and overrides the `insecure` setting in the `tls`.
+<2> The client side TLS configuration. Defines paths to TLS certificates.
+<3> Headers are sent in every HTTP request.
+
+==== Jaeger exporter
+
+The Jaeger exporter exports data using the Jaeger proto format through gRPC.
+
+* Support level: link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]
+* Supported signals: traces
+
+.OpenTelemetry Collector custom resource with enabled Jaeger exporter
+[source,yaml]
+----
+  config: |
+    exporters:
+      jaeger:
+        endpoint: jaeger-all-in-one:14250 <1>
+        tls: <2>
+          ca_file: ca.pem
+          cert_file: cert.pem
+          key_file: key.pem
+    service:
+      pipelines:
+        traces:
+          exporters: [jaeger]
+----
+<1> The Jaeger gRPC endpoint.
+<2> The client side TLS configuration. Defines paths to TLS certificates.
+
+==== Logging exporter
```

Comment:
[Anchoring in module files](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-module-files)
Missing anchor IDs

Reply:
Thank you for the link. All anchor IDs are now added in this file. I will check the other files too.